src/snarky.d.ts

```diff
@@ -257,11 +536,15 @@ export class Circuit {
 
   static verify(publicInput: any[], vk: VerificationKey, pi: Proof): boolean;
 
-  static toFieldElements<A>(A): Field[];
+  static toFields<A>(a: A): Field[];
+
+  static inProver(): boolean;
+
+  static inCheckedComputation(): boolean;
```

Comment:
Just curious - what's the difference between `inProver` and `inCheckedComputation`?